index.html

```diff
@@ -3597,6 +3629,33 @@ <h3>Data Validation</h3>
 									error</a>, set to "<code>ltr</code>".</p>
 						</li>
 
+						<li id="validate-resources">
+							<p>(<a href="#resource-list"></a>) If <var>data["resources"]</var> is set, let
+									<var>uniqueURLs</var> be an empty <a
+									href="https://infra.spec.whatwg.org/#ordered-set">set</a>. <a
+									href="https://infra.spec.whatwg.org/#list-iterate">For each</a>
```

Comment:
Shouldn't this say 'ordered set'? This is the term used elsewhere. In my reading, the infra standard term 'set' and 'ordered set' seem to be synonymous (the section head says 'set' and the content refers to ordered sets; I would think this is an editorial bug in the infra standard) but we should be consistent...